jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BaseVisualizationModel.java

```diff
@@ -130,6 +105,34 @@ public BaseVisualizationModel(
     }
   }
 
+  private LayoutModel<N, Point2D> createLayoutModel(Network<N, E> network, Dimension layoutSize) {
+    String modelStructure = System.getProperty("layout.model");
```

Comment:
Why do this via a system property, rather than doing configuration the same way that we do all the rest (i.e., constructor argument which is a ModelStructure)?

Reply:
Do we want to persist the model structure anywhere (e.g. as a field in BaseVisualizationModel)?  Should it be settable?

Reply:
I don't want to expose it that much yet, and I want to be able to run the same already compiled program with different spatial data structures.